engine/helpers.go

```diff
@@ -185,19 +185,27 @@ func (bot *Engine) SetSubsystem(subSystemName string, enable bool) error {
 	case SyncManagerName:
 		if enable {
 			if bot.currencyPairSyncer == nil {
-				exchangeSyncCfg := &SyncManagerConfig{
-					SynchronizeTicker:       bot.Settings.EnableTickerSyncing,
-					SynchronizeOrderbook:    bot.Settings.EnableOrderbookSyncing,
-					SynchronizeTrades:       bot.Settings.EnableTradeSyncing,
-					SynchronizeContinuously: bot.Settings.SyncContinuously,
-					TimeoutREST:             bot.Settings.SyncTimeoutREST,
-					TimeoutWebsocket:        bot.Settings.SyncTimeoutWebsocket,
-					NumWorkers:              bot.Settings.SyncWorkersCount,
-					FiatDisplayCurrency:     bot.Config.Currency.FiatDisplayCurrency,
-					Verbose:                 bot.Settings.Verbose,
+				cfg := bot.Config.SyncManagerConfig
+				cfg.SynchronizeTicker = bot.Settings.EnableTickerSyncing && cfg.SynchronizeTicker
```

Comment:
When ticker sync is enabled in config this pattern used to override if set using `flagSet.WithBool` functionality in engine.
```
.\gocryptotrader -orderbooksync=true -tickersync=false
```

Reply:
Could you rephrase that please?

Reply:
When ticker sync is set to true in config, when I try to set it to false via command line it will not. I don't english well.

Reply:
For the inverse if I have both set to false in config and I run this command, it doesn't work:

```
 .\gocryptotrader --orderbooksync=true --tickersync=true
```

Reply:
Default app param values have now become `false` to address this

Reply:
Resolved, I forgot about the lovely existing helper functions